benchmarks/automated/README.md

```diff
@@ -145,34 +142,44 @@ The final benchmark report will be available in markdown format as `report.md` i
 ## Features of the automation:
 1. To save time by *not* creating new instances for every benchmark run for local testing, use the '--do-not-terminate' flag. This will automatically create a file called 'instances.yaml' and write instance-related data into the file so that it may be re-used next time.
 ```
-python test/benchmark/run_benchmark.py --do-not-terminate
+python benchmarks/automated/run_benchmark.py --do-not-terminate
 ```
 
 2. To re-use an instance already recorded in `instances.yaml`, use the '--use-instances' flag:
 ```
-python test/benchmark/run_benchmark.py --use-instances <full_path_to>/instances.yaml --do-no-terminate
+python benchmarks/automated/run_benchmark.py --use-instances <full_path_to>/instances.yaml --do-no-terminate
```

Comment:
How many EC2 instances are used in the test?

Reply:
@nskool Also, as we discussed seems this option would not work with the refactor and yaml files.